hashicorp/dev-portal
# chore: tweaks to search modal

## 🔗 Relevant links

- [Preview link][preview] 🔎
- [Asana task][task] 🎟️

## 🗒️ What

Makes tweaks to the command bar modal (aka search modal) appearance. Specifically, tweaks the background color, input font weight, and padding around the modal header area.

## 📸 Design Screenshots

### Background color

| Mode | Before (`primary`) | After (`faint`) |
| - | - | - |
| Light | <img alt="background-light-primary" width="300" src="https://github.com/hashicorp/dev-portal/assets/4624598/1393892a-3bd4-4755-bae2-aead3840201f" /> | <img alt="background-light-faint" width="300" src="https://github.com/hashicorp/dev-portal/assets/4624598/deea887d-9df3-443a-ac2b-a565ccc423a6" /> |
| Dark | <img alt="background-dark-primary" width="300" src="https://github.com/hashicorp/dev-portal/assets/4624598/2dcd2230-3255-471e-9952-4e94718420da" /> | <img alt="background-dark-faint" width="300" src="https://github.com/hashicorp/dev-portal/assets/4624598/a57f620b-dffd-468e-b622-39bc7229c115" /> |

### Input font weight

| Before (`medium`) | After (`regular`) |
| - | - |
| ![input-font-medium](https://github.com/hashicorp/dev-portal/assets/4624598/eceea648-7d06-4f86-a206-d9a585472eba) | ![input-font-regular](https://github.com/hashicorp/dev-portal/assets/4624598/92a4535a-8f83-4fc0-b8ec-909a605b9641) |

### Modal header padding

| Before (`12px`) | After (`16px 20px`) |
| - | - |
| ![modal-heading-padding-12](https://github.com/hashicorp/dev-portal/assets/4624598/210e5a0c-34a5-4868-b878-77f7623b172b) | ![modal-heading-padding-16-20](https://github.com/hashicorp/dev-portal/assets/4624598/f359347f-9de3-41d5-b4f2-9d1f6763828a) |

## 🧪 Testing

- [ ] Visit the [preview]
    - Ensure the revised background color is as expected, in both light and dark mode, including hover states for items on that background
    - Ensure the revised font-weight is as expected
    - Ensure the modal padding is as expected

## 💭 Anything else?

Not at the moment!

[preview]: https://dev-portal-git-zssearch-modal-tweaks-hashicorp.vercel.app
[task]: https://app.asana.com/0/1204333060736722/1204910640074751/f
